nebari_workflow_controller/utils.py

```diff
@@ -45,6 +48,28 @@ def sent_by_argo(workflow: dict):
     return None
 
 
+def valid_argo_roles():
+    # TODO: determine a more extensible way of generating a list of valid roles
+    return ["argo-admin", "argo-developer"]
```

Comment:
I think this should be a config option somehow.  Most other things in Nebari use traitlets, but I'd feel okay using an VALID_ARGO_ROLES ENV var for now.

Reply:
Agreed, this is not a good solution. I added a `configmap` to the nebari-workflow-controller deployment that provides a list of valid Argo roles ([here](https://github.com/nebari-dev/nebari/pull/1832/commits/5e5aff2ae6d32360d60ffd8a922c8dc6d1244e0f)). How does that solution work for you?